appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -118,6 +118,22 @@
 		return createOKResponse(userService.getPublicUser(id), false);
 	}
 
+	@RequestMapping(value = "/{id}/vpnAuthCertificate.crt", produces = { "application/x-x509-user-cert" })
+	public @ResponseBody byte[] getVpnAuthCertificate(@RequestHeader(value = Constants.PASSWORD_HEADER) Optional<String> password,
+			@PathVariable UUID id)
+	{
+		return CryptoSession.execute(password, () -> userService.canAccessPrivateData(id),
+				() -> userService.getPrivateUser(id).getPrivateData().getVpnProfile().getVpnAuthCertificateByteArray());
+	}
+
+	@RequestMapping(value = "/{id}/apple.mobileconfig", produces = { "application/x-apple-aspen-config" })
+	public @ResponseBody byte[] getVpnAppleMobileConfig(
```

Comment:
My understanding is that this is a text file, so ``String`` would be a more appropriate return type. We can do this at the time we finalize the iOS-related code.